public/static/locales/fr/form-builder.json

```diff
@@ -43,7 +43,16 @@
   "formSubmissionDisabledInPreview": "[FR] Form submission is disabled in preview",
   "formTitle": "[FR] Form title",
   "french": "Français",
-  "gcForms": "[FR] GC Forms",
+  "gcForms": "Formulaires GC",
+  "gcFormsStart": "Créer",
+  "gcFormsEdit": "Editer",
+  "gcFormsPreview": "Prévisualiser",
+  "gcFormsResponseDelivery": "Livraison des réponses",
+  "gcFormsTranslate": "Traduction",
```

Comment:
@ioanacontu suggested (on Slack) using `Traduire` instead of `Traduction` here. I do agree with her.